addons/portal/wizard/portal_wizard.py

```diff
@@ -40,23 +40,37 @@ def _default_user_ids(self):
                 # make sure that each contact appears at most once in the list
                 if contact.id not in contact_ids:
                     contact_ids.add(contact.id)
-                    in_portal = False
-                    if contact.user_ids:
-                        in_portal = self.env.ref('base.group_portal') in contact.user_ids[0].groups_id
                     user_changes.append((0, 0, {
                         'partner_id': contact.id,
                         'email': contact.email,
-                        'in_portal': in_portal,
                     }))
         return user_changes
 
     user_ids = fields.One2many('portal.wizard.user', 'wizard_id', string='Users',default=_default_user_ids)
     welcome_message = fields.Text('Invitation Message', help="This text is included in the email sent to new users of the portal.")
 
-    def action_apply(self):
-        self.ensure_one()
-        self.user_ids.action_apply()
-        return {'type': 'ir.actions.act_window_close'}
+    @api.model
+    def action_open_wizard(self):
```

Comment:
See explanation above, we need the record to exist to be able to execute an action button on it (that's why we first create the record and then open the form view, instead of opening the form view with default value)

Reply:
Indeed, clearer that way. I did not understand it was the lines buttons that was the issue. You could be clearer and explain that o2m sub records have to exist to call action on them through buttons, hence server action.